src/test/java/io/supertokens/test/session/api/RefreshSessionAPITest2_7.java

```diff
@@ -363,6 +364,7 @@ public void successOutputWithValidRefreshTokenTest() throws Exception {
 
         String userId = "userId";
         JsonObject userDataInJWT = new JsonObject();
+        userDataInJWT.add("nullProp", JsonNull.INSTANCE);
```

Comment:
you should check that post refresh (in this test), that this still has the `nullProp` key / value

Reply:
`checkRefreshSessionResponse` called on 393 already checks this.